packages/llama-cpp/build.sh

```diff
@@ -0,0 +1,23 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/ggerganov/llama.cpp
+TERMUX_PKG_DESCRIPTION="Port of Facebook's LLaMA model in C/C++"
+TERMUX_PKG_LICENSE=GPL-3.0
+TERMUX_PKG_MAINTAINER=@termux
+TERMUX_PKG_VERSION=294f424
+TERMUX_PKG_SRCURL=$TERMUX_PKG_HOMEPAGE/archive/master-$TERMUX_PKG_VERSION.tar.gz
+TERMUX_PKG_SHA256=95effaa75fdf1e7fb4819500f3aa6a9c970dbe36392a51a4ead904660841cd93
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="openmpi, libopenblas"
+TERMUX_PKG_RECOMMENDS="python-numpy, python-sentencepiece"
+TERMUX_PKG_EXTRA_CONFIGURE_ARGS="
+-DCMAKE_INSTALL_PREFIX=$TERMUX_PREFIX
```

Comment:
This option is also not needed. It will be set automatically by the building system.

Reply:
I also don't think it is good to install all the binary files to `$TERMUX_PREFIX`. This package contain some binary like `bin/benchmark` and its name is so simple that it may conflict with other packages...